hl7-parser/src/main/java/gov/cdc/dataingestion/hl7/helper/integration/HL7Parser.java

```diff
@@ -51,60 +50,29 @@ public String processFhsMessage(String message) {
     }
 
     public String hl7ORUValidation(String message) throws DiHL7Exception {
-        /**
-         * Default validation include
-         * - number of sensible
-         * - max length on string type
-         * - format for telephone and timestamp, etc
-         * */
-        defaultOruR01Validator(message);
 
         /**
-         * This goes in-dept into message structure and validate based on the custom definition
+         Rule can be expanded if guideline is specified in the future - for now this is just the template
+         - Only enforcing Phone Number Rule
          * */
-        customOruR01Validator(message);
+        oruR01Validator(message);
 
         return message;
     }
 
-    private void defaultOruR01Validator(String message) throws DiHL7Exception {
-        this.context.setValidationContext(ValidationContextFactory.defaultValidation());
-        PipeParser parser = context.getPipeParser();
-        try {
-            parser.parse(message);
-        } catch (HL7Exception e){
-            throw new DiHL7Exception(EX_MESSAGE + e.getMessage());
-        }
-
-        // Ignore sonar queue complain as this is coming from Library
-        ValidationRuleBuilder builder = new DefaultValidationBuilder() { // NOSONAR
-            @Override
-            protected  void configure() {
-                super.configure();
-                forAllVersions().message(ORU, ORU_01).onlyKnownSegments();
```

Comment:
this is the check kept failing the message with sft and spm segment

Reply:
perhaps it was the message she sent in at the time, but I was debugged this with her and able to reproduced it even in my local environment.

it had something to do with the escape characters '\r' and '\n', only in some specific scenario where input is appended with '\n' and di replace '\n' with '\r' (this some how cause the onlyKnowSegment check to not recognized the SFT and SPM segment

Reply:
also, additional info for this whenever it occurred in test.
I set the validation header back to false and the message was able to pass without throw the exception. This header basically force the message flow through the check above which in the failed ingestion